<launch>

<!-- imu -->
  <node pkg="microstrain_3dm_gx3" type="imu_node" name="microstrain_3dm_gx3_node" output="screen">
    <param name="time_offset" value="0" />
    <param name="port"  type="string" value="/dev/ttyACM1" />
    <param name="autostart" type="bool" value="true" />
    <param name="frameid"  type="string" value="imu" />
    <param name="autocalibrate" type="bool" value="false" />
    <param name="angular_velocity_stdev" type="double" value="0.00017" />
  </node>

  <node pkg="microstrain_3dm_gx3" type="imu2pose.py" name="imu2pose" output="screen"/>

</launch>
